Framework/NexusGeometry/test/NexusGeometrySaveTest.h

```diff
@@ -1111,6 +1112,56 @@ Instrument cache.
     TS_ASSERT_THROWS(tester.openfullH5Path(transformationsPath),
                      H5::GroupIException &)
   }
+
```

Comment:
Would be better if you had `TDD` built up a test suite. There are lots of functions that could have been designed up front, for example, you have functions that go from the shapeinfo format cylinder to the nexus 3 vertex format.